src/coreWebVitals.ts

```diff
@@ -0,0 +1,121 @@
+import type { ReportHandler } from 'web-vitals';
+import { getCLS, getFCP, getFID, getLCP, getTTFB } from 'web-vitals';
+
+type CoreWebVitalsPayload = {
+	page_view_id: string | null;
+	browser_id: string | null;
+	fid: null | number;
+	cls: null | number;
+	lcp: null | number;
+	fcp: null | number;
+	ttfb: null | number;
+};
+
+const coreWebVitalsPayload: CoreWebVitalsPayload = {
```

Comment:
It might be worth giving this a different name for clarity but I don't have a great suggestion.
Maybe the type is `CoreWebVitalsPayload` and the const is `pageCoreWebVitals`?

Reply:
I think the `camelCase: PascalCase` notation for variables and their types is quite idiomatic in JS and Scala codebases.